ajax/libs/simplebar/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "simplebar",
+  "filename": "simplebar.js",
+  "description": "Scrollbars, simpler.",
+  "homepage": "https://grsmto.github.io/simplebar/",
+  "keywords": [
+    "simplebar",
+    "scrollbar"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/grsmto/simplebar.git"
+  },
+  "author": "Adrien Grsmto from a fork by Jonathan Nicol",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/grsmto/simplebar.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
Let's use `"**/!(*.[0-9])*"` because we should exclude the filename with version number such as `1.0`(simplebar.1.0.0.css, simplebar.1.0.0.min.js) and `1.1`(simplebar.1.1.0.css, simplebar.1.1.0.min.js).
Thank you.